systemd/authd.service

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=Authd daemon service
+After=authd.socket
+Requires=authd.socket
```

Comment:
I see the definition of PartOf slightly changed over the years and some units have been migrated to Requires + PartOf instead of only PartOf. Mind changing adsys to that model too?